examples/cd_context_ecg.ipynb

```diff
@@ -0,0 +1,1039 @@
+{
```

Comment:
<p>'More generally, <strong>the context-aware drift detector can detect changes in the data distribution which cannot be attributed to a change in the context variable, which we deem permissible' </strong>- This reads like the changes that cannot be attributed are deemed permissible. Maybe instead:</p><p>'More generally, <strong>the context-aware drift detector detects changes in the data distribution which cannot be attributed to a permissible change in the context variable.' </strong></p><p> </p><p>Some of my comments from the start of the other notebook also apply here.</p><p> </p><p>'The drift detector will be conditioned on the ECG classifier model predictions, cluster membership by an unsupervised clustering algorithm and time. ' - Is this describing three different context variables that will be demonstrated? Not sure if clear.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi-detect/pull/461/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>